CMakeLists.txt

```diff
@@ -0,0 +1,92 @@
+cmake_minimum_required(VERSION 3.13)
+cmake_policy(VERSION 3.13)
+
+project(fty-metric-store
+    VERSION 1.0.0
+    DESCRIPTION "Persistance for metrics"
```

Comment:
persistence

Reply:
Sorry copy of description of the project description ...